lib/ddtrace/ext/app_types.rb

```diff
@@ -6,6 +6,7 @@ module AppTypes
       CACHE = 'cache'.freeze
       WORKER = 'worker'.freeze
       CUSTOM = 'custom'.freeze
+      TEST = 'test'.freeze
```

Comment:
I'm assuming this new value is ok with the agent and/or backend teams, right?

Reply:
Yes. It is the same value across `dd-trace-java`, `dd-trace-py`, and `dd-trace-dotnet`.